packages/pigeon/pigeons/core_tests.dart

```diff
@@ -50,23 +116,13 @@ abstract class HostIntegrationCoreApi {
   @ObjCSelector('echoAllTypes:')
   AllTypes echoAllTypes(AllTypes everything);
 
+  /// Returns the passed object, to test serialization and deserialization.
+  @ObjCSelector('echoAllNullableTypes:')
+  AllNullableTypes echoAllNullableTypes(AllNullableTypes everything);
+
   /// Returns an error, to test error handling.
   void throwError();
 
-  /// Returns the inner `aString` value from the wrapped object, to test
-  /// sending of nested objects.
-  @ObjCSelector('extractNestedStringFrom:')
-  String? extractNestedString(AllTypesWrapper wrapper);
-
-  /// Returns the inner `aString` value from the wrapped object, to test
-  /// sending of nested objects.
-  @ObjCSelector('createNestedObjectWithString:')
-  AllTypesWrapper createNestedString(String string);
-
-  /// Returns passed in arguments of multiple types.
-  @ObjCSelector('sendMultipleTypesABool:anInt:aString:')
-  AllTypes sendMultipleTypes(bool aBool, int anInt, String aString);
```

Comment:
Why change this one? Since what we want to test here is multiple arity, adding nullability seems like unnecessary complexity.

Reply:
Mainly to avoid instantiating full classes with tons of values that aren't being used across all languages. I can "change it back" (although it was never technically different) if that's more correct in your opinion

Reply:
Oh right, I forgot how messy that is since we don't have defaults. Good call.